ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -67,7 +83,7 @@
 - job-template:
     name: 'ceph-ansible-prs-{release}-{ansible_version}-{scenario}'
     id: 'ceph-ansible-prs-auto'
-    node: vagrant&&libvirt
+    node: '{slave_labels}'
```

Comment:
Does this actually work? I couldn't get it to work when I tried this for ``only-trigger-phrase``.

Reply:
It does.  Kind of surprised me too.  Works on my workstation running jjb v2.0.0.

Reply:
Are we running that version when we update the jenkins jobs? If this does work it'd be nice to get rid of the extra job template. Maybe the trouble I had with ``only-trigger-phrase`` was that it was a boolean.

Reply:
https://github.com/ceph/ceph-build/blob/master/jenkins-job-builder/build/build#L12

```
dgalloway@w541 ~ ()$ pip list | grep jenkins-job-builder
jenkins-job-builder (2.0.0.0b2)
```